cg/cli/workflow/commands.py

```diff
@@ -112,9 +119,32 @@ def store_available(context: click.Context, dry_run: bool) -> None:
         raise click.Abort
 
 
+@click.command("clean-rsync-dirs")
```

Comment:
perhaps thematically better to move it to cli/clean? The options are already available there